versions/latest/modules/en/pages/cluster-deployment/enable-cluster-agent-scheduling-customization.adoc

```diff
@@ -0,0 +1,74 @@
+= Enabling Cluster Agent Scheduling Customization
+
+In Rancher v2.11.0 and later, you can enable the automatic deployment of a Priority Class and Pod Disruption Budget for the `cattle-cluster-agent`.
```

Comment:
```suggestion
The `cattle-cluster-agent` allows enabling automatic deployment of a Priority Class and Pod Disruption Budget.
```

This is something I should have caught on the Community PR, we should reword this to drop the introductory phrase. Calling out the version (e.g. available .0 and later) for something that's available for all patch versions in the minor is redundant and in the case of our product docs .0 has no relevancy since it's not a Prime release.